### PR TITLE
add more configurations; add statistics webserver

### PR DESCRIPTION
**Pull request by bladedoyle**
Enhanced the configuration.
Added more config options for p2pool, monero, and xmrig
Added a p2pool server statistics webserver container that is activated by "Enable Server Statistics" option (default is enabled).
A few random fixes and README.md updates.

Note: This is the extent of the options I plan to add (unless there are specific requests, or p2pool adds new options).

### CODE REVIEW

**Comment by Titaniumtown** [comment]
Why is the statistics webserver only for docker....?

**Comment by bladedoyle** [comment]
@Titaniumtown 
Because this version of the statistics webserver is written in Python integration into the p2pool C++ codebase is not possible.

I could make it easy to use the statistics webserver with a non-dockerized p2pool instance if there is interest and SChernykh agrees.  This would involve adding code to make it a python package, and pushing releases to PyPi for easy installation.  If it were not exclusive to docker-compose we would want to move the code up into the project root (p2pool/statistics-webserver or similar), and add instructions to the main README.

Im happy to work on that if there is interest.

**Comment by bladedoyle** [comment]
image

**Comment by bladedoyle** [comment]
Good Grief.  I still cant figure out how to use git properly.  
I needed to make a change to the mount attribute for p2pool data into statistics container and made a bit of a mess as usual.
Anyway, I think I finally got this branch updated with master and the change I needed is added.

File: docker-compose/statistics/app/templates/index.html [hunk 1]
```diff
@@ -0,0 +1,171 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css"
```
Comment (SChernykh) [inline docker-compose/statistics/app/templates/index.html:6]: Can we have all `.css` and `.js` files locally? This page doesn't need to depend on other web servers, which can go down or track the user.

**Comment by bladedoyle** [comment]
Oops, one small change needed for default configuration - should not include my own rpc login :)

**Comment by bladedoyle** [comment]
I am removing the health check from the monero container since:
1) you fixed the p2pool hangs on startup when monero rcp isnt listening issue
and
2) too many syslog messages: https://github.com/docker/for-linux/issues/679

**Comment by bladedoyle** [comment]
healthcheck removal complete.  Ready to merge.

**Pull request merged**